website/docs/language/state/encryption.html.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: "language"
+page_title: "State: Client-Side Remote State Encryption"
+sidebar_current: "docs-state-encryption"
+description: |-
+  Client-Side Remote State Encryption.
+---
+
+# Client-Side Remote State Encryption
+
+State regularly contains mission-critical sensitive information such as credentials.
+
+Several remote state backends already support server-side encryption, but
+
+- many do not
+- some implementations effectively turn encryption into nothing but a proxy for key access control  
+- you need to trust the remote backend operator
+- you need to trust your communication channel to the remote backend
+
+This **experimental feature** lets you encrypt the complete state on the client before transferring it.
+
+Depending on your choice of state crypto provider, no third party ever sees the key.
+
+## Limitations
+
+Client-side state encryption will not work with [enhanced backends](/docs/language/settings/backends/index.html), as 
+those need access to the information in the state to function correctly.
+
+**Client-side remote state encryption is an experimental feature!** Implementation and configuration details are subject to change, 
+and not all remote state backends have been fully tested!
+
+## State Crypto Providers
+
+State crypto providers are a transparent encryption layer around the communication with remote state backends.
+
+**This is an experimental feature!**
+
+You can configure state crypto providers by using two environment variables:
+
+- `TF_REMOTE_STATE_ENCRYPTION`: configuration used for encryption and decryption
+- `TF_REMOTE_STATE_DECRYPTION_FALLBACK`: fallback configuration for decryption, tried if decryption fails with the first choice  
+
+You set each of these to a json document with two fields
+
+  * `implementation`: select a _state crypto provider_ by name
+  * `parameters`: configure the _state crypto provider_
+
+to enable client-side remote state encryption. To disable, either do not set the variable at all or set it to a blank value.
+
+Right now, there is only one value for `implementation`: `client-side/AES256-cfb/SHA256`. 
```

Comment:
Cosmetic, I would also capitalize "cfb", if you're capitalizing aes and sha.

Reply:
Done!